### ISSUE TITLE
Python executable in installed files should be supplied by the caller

### ISSUE THREAD

**Issue by yurivict** [electronic-structure/SIRIUS#758]
For example in ```apps/upf/upf_to_json``` ```python``` should be replaced with cmake's variable ```PYTHON3_COMMAND``` or similar.

**Comment by simonpintarelli** [comment]
At the moment SIRIUS only depends explicitly on python when the python module is built (cf https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/sirius/package.py). In most cases the python module isn't required. Spack has become the preferred way for installing SIRIUS. On HPC systems this would then often require to build python from source, if we would make it a hard dependency.
The typical user will run the conversion script locally (input file creation) and later run the computation on a cluster. The typical end user will most likely not compile sirius locally, but run the conversion script directly from the source repo.

Is there no python exectuable called `python` on freebsd? 

We have the conversion script in a separate repository. [aiida](https://github.com/aiidateam/aiida-core) needs it to be installable via `pip`. The release on pypi is currently a python module only https://pypi.org/project/upf-to-json, without an executable. 
I've added a python click interface https://github.com/simonpintarelli/upf_to_json/tree/click, I can make a release of this one on pypi if that would help. Then we could also add an option to cmake, to turn off the installation of the `upf_to_json` script (which seems to be broken on freebsd due to name of the python exectuable).

**Comment by yurivict** [comment]
> Is there no python exectuable called python on freebsd?

No. Multiple Python versions can be installed concurrently, each having its unique name/path.
FreeBSD port has some Python version as default and it has the ```PYTHON_CMD```  variable containing the path.
